client/main.lua

```diff
@@ -83,19 +83,20 @@ function OpenPhone()
         if HasPhone then
             PhoneData.PlayerData = QBCore.Functions.GetPlayerData()
             SetNuiFocus(true, true)
-            -- SetNuiFocusKeepInput(true)
             SendNUIMessage({
                 event = 'z-phone',
                 isOpen = true,
             })
             PhoneData.isOpen = true
-
+	if Config.AllowWalking then
+        SetNuiFocusKeepInput(true)
```

Comment:
what happens if you open your phone and press the A/W/S/D keyboard? will the character move too?

Reply:
whe  you press A/W/S/D only your character move, and you can still moving the cursor on the phone